pkg/config/filter.go

```diff
@@ -3,33 +3,64 @@ package config
 import (
 	"fmt"
 	"os"
+	"path/filepath"
 	"strings"
 
 	"github.com/gobwas/glob"
 
 	"github.com/open-policy-agent/opa/bundle"
-	"github.com/open-policy-agent/opa/loader"
 )
 
-func FilterIgnoredPaths(paths, ignore []string, checkFileExists bool) ([]string, error) {
-	policyPaths := paths
+type DirEntryFilter = func(path string, info os.DirEntry, err error) error
```

Comment:
[nit] DirEntryFilter doesn't need to be exported. I wonder if it even warrants its own type, but it won't hurt.